### DISCUSSION TITLE
silent installing and close chatty with mpv

### DISCUSSION THREAD

**Discussion by super-lurker** [streamlink/streamlink#3695] [Q&A] [answered]
how do i silent install streamlink and how do I make chatty close when mpv closes?

**Comment by gravyboat** [comment]
> how do i silent install streamlink

This is not something we're going to support. If you're writing some sort of third party script to do that you'll have to figure it out.

> how do I make chatty close when mpv closes

This isn't related to Streamlink at all, you'll have to write the code to handle this.

**Comment by bastimeyer** [comment, marked as answer]
> silent install

I assume you are talking about the Windows installer. This is a regular NSIS installer, so all the supported NSIS parameters are supported.
- https://github.com/streamlink/streamlink/blob/master/script/makeinstaller.sh
- https://nsis.sourceforge.io/Which_command_line_parameters_can_be_used_to_configure_installers%3F

> how do I make chatty close when mpv closes?

This depends on how you've started Streamlink. But that is a problem you have to solve on your own.
